src/NodaTime.Test/NodaTime.Test.csproj

```diff
@@ -9,6 +9,8 @@
     <Deterministic>True</Deterministic>
     <IsPackable>False</IsPackable>
     <DefineConstants Condition="$(TargetFramework.Contains('netcore'))">NETCORE</DefineConstants>
+    <ApplicationIcon />
```

Comment:
I'll remove these changes on this branch - there's no need for this file to change at all.